src/erlcloud_s3.erl

```diff
@@ -560,11 +560,11 @@ get_notification_filter(Path, Config) ->
 get_notification_filter_do([]) -> [];
 get_notification_filter_do(S3Key) ->
     #xmlElement{content = Content} = hd(S3Key),
-    [[{name, "Prefix"}, {value, Prefix}],
-     [{name, "Suffix"}, {value, Suffix}]] =
-    [erlcloud_xml:decode([{name, "Name", text}, {value, "Value", text}], C)
-     || C <- Content],
-    {filter, [{prefix, Prefix}, {suffix, Suffix}]}.
+    GV = fun(Key, PL) -> proplists:get_value(Key, PL) end,
```

Comment:
okay, this is purely code stylish and not functional but IMO this unnecessary and does not make code shorter. 
it just makes Filter comprehension below fit into single line of 80 chars and nothing else. 

given that the library does not have strict code style requirements this is fine to go.
give me some time if I can come up with a better formatting of the same functionality before merging